x-pack/legacy/plugins/siem/public/containers/helpers.ts

```diff
@@ -4,12 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { FetchPolicy } from 'apollo-client';
+import { FetchPolicy } from '@apollo/client';
 import { isString } from 'lodash/fp';
 
 import { ESQuery } from '../../common/typed_json';
 
 export const createFilter = (filterQuery: ESQuery | string | undefined) =>
   isString(filterQuery) ? filterQuery : JSON.stringify(filterQuery);
 
-export const getDefaultFetchPolicy = (): FetchPolicy => 'cache-and-network';
+export const getDefaultFetchPolicy = (): FetchPolicy => 'cache-first';
```

Comment:
did the meaning change? because when you pick `cache-first`, all data are cache and we never get back new ones but with `cache-and-network`, it will be cache and then getting the latest and see if it needs to update the cache